tests/integration_tests/functional/test_signals.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests scenarios for Firecracker signal handling."""
+
+import os
+from signal import SIGBUS, SIGSEGV
+from time import sleep
+
+import pytest
+
+import host_tools.logging as log_tools
+
+
+@pytest.mark.parametrize(
+    "signum",
+    [SIGBUS, SIGSEGV]
+)
+def test_sigbus(test_microvm_with_api, signum):
```

Comment:
nit: this test is called for both `sigbus` and `sigsegv` yet it is named `test_sigbus`